src/main/webapp/mapScript.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Initialize the map to show Mountain View and events in that area
+ */
+function initMap() {
+
+  // Coordinates for the default map center of Mountain View, CA
+  const mapViewDefault = {lat: 37.3868, lng: -122.085}; 
+
+  const map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 8,
```

Comment:
what's the significance of 8?

Reply:
I just set the zoom to 8 to be the default value to test the map, but it will be changed when I add the functionality to show a certain radius.

Reply:
Screenshot of UI: https://screenshot.googleplex.com/kGqWj8HAkqx

Reply:
ok that's fine! Can you add a quick comment explaining?

Also thanks for the screenshot! For future comments though, we'll want to avoid using the screenshot extension links because even though other users will be blocked from seeing it, it's still internal :)